packages/flutter_tools/lib/src/commands/build_aot.dart

```diff
@@ -356,7 +357,7 @@ Future<String> _buildAotSnapshot(
     final Set<String> snapshotInputPaths = await readDepfile(dependencies)
```

Comment:
(I can't add a comment to the appropriate lines in this file)

Maybe add a todo in this file where gen_snapshot is referenced to indicate that this should be using the new snapshotter instead?

Reply:
Good point! Done.